<?xml version="1.0"?>
<launch>
  <node pkg="ig_active_reconstruction_ros" type="basic_view_planner" name="basic_view_planner" clear_params="true" output="screen">
    
    <param name="discard_visited" value="true" />
    <param name="max_visits" value="-1" />
    <param name="cost_weight" value="10" />
    <param name="max_calls" value="20" />
    <!-- rosparam param="ig_names">[OcclusionAwareIg, UnobservedVoxelIg, RearSideVoxelIg, RearSideEntropyIg, ProximityCountIg, VasquezGomezAreaFactorIg, AverageEntropyIg]</rosparam -->
    <rosparam param="ig_names">[OcclusionAwareIg]</rosparam>
      <!--<rosparam param="ig_weights">[1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0]</rosparam>-->
      <rosparam param="ig_weights">[1.0]</rosparam>
    
  </node>
</launch>
